### PR TITLE
Remove tracked build files in `output/`

### PR DESCRIPTION
**Pull request by ewels**
The `output/` directory contains the built assets created by jbake. It is [ignored in `.gitignore`](https://github.com/nextflow-io/website/blob/01256fa1acc0b5fb282182e8a0ac94dc2bf69720/.gitignore#L2). However, a bunch of files are still tracked in git, presumably from before this was added. This means that during development the `git status` always shows a huge list of changes whenever anything is changed. These have to be ignored.

Here I've removed these files from git (and removed the duplicate `.gitignore` entry). This should not affect deployments, as the build and upload does not rely on files being tracked in git.

@pditommaso - please check that I'm correct in this 😅

### CODE REVIEW

**Comment by pditommaso** [comment]
Do tell the truth, don't know. In theory, should be all file generated from the template or copied by assets. But.. does it hurt? I'd keep until we don't migrate to the new web site

**Comment by pditommaso** [comment]
* mildly irritating * 😄 I'll double-check in my local copy the exact content

**Comment by pditommaso** [comment]
Ok, I've found only a few differences, but I guess those files are not used (in any case I've made a backup)

Screenshot 2022-12-14 at 18 53 20

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,3 @@ output/
 assets/docs/latest/en/
 stuff/
 .DS_Store
-output
```
Comment (pditommaso) [inline .gitignore:6]: think this should stay, because those files should not be committed, no?

Comment (ewels) [reply .gitignore:6]: It's a duplicate - there's another line further up the file already.

Comment (pditommaso) [reply .gitignore:6]: 😮

**Review by pditommaso — APPROVED** [review]

**Comment by ewels** [comment]
Ahhhh that's better 😆  Thank you!

image

94f

**Pull request merged**